NASA Challenge_[@elementrobotics]_[LunarSim] (closes space-ros/demos#50)

mazm0002 opened the pull request:
Depends on space-ros/demos#51

This PR adds instructions to run the spaceROS Lunar Sim demo as well as extensive documentation on how users can integrate their own Digital Elevation Models (DEMs), Ephemeris data and textures.

mazm0002 committed:
Added page in docs for lunar sim instructions (#44).

The demonstrations now include a lunar simulation.

This commit adds a documentation page to the website describing how the
simulation can be built and run.

ivanperez-keera commented:
Hi @mazm0002 . I'd like to prepare this issue for merging together with the demo.

I tried to make the following changes to make the PR comply with our process.

Can you please:

- Squash all commits.
- Modify the commit message of the only commit (after squash) to the following:
```
Added page in docs for lunar sim instructions (#44).
    
The demonstrations now include a lunar simulation.
    
This commit adds a documentation page to the website describing how the
simulation can be built and run.
```

mazm0002 commented:
Hi @ivanperez-keera, thanks for helping in getting this merged! I've squashed the commits and changed the message to the suggested one. Let me know if anything else needs to be fixed here or with the demo.

ivanperez-keera commented:
Thank you @mazm0002  for acting so quickly on this. I'll review the associated PR in the demos repo and post any other requests for changes there.

It makes sense to merge things first in the demos repo and then here, just so that we don't have documentation that points to a demo that is not merged yet.

mazm0002 committed:
Changed project name to lunar_terrain (#44)
